src/components/VueHotelDatepicker.vue

```diff
@@ -3,7 +3,8 @@
     <input v-model="value"
            :placeholder="placeholder"
            type="text" class="vhd-input" aria-label="vue-hotel-datepicker-input"
-           @click="active = !active">
+           @mousedown.prevent="toggle"
+           @focus.prevent="toggle">
```

Comment:
```suggestion
           @focus.prevent="toggle"
           @blur.prevent="toggle">
```
i guess it need to close `onblur`. (maybe it will open next calendar input)

Reply:
The issue with blur is, that when you tab into the input to open the calendar it closes as soon as you try to select the dates because the input loses it's focus.